docs/manual/common/concepts/ServiceDiscovery.md

```diff
@@ -0,0 +1,39 @@
+# Service Discovery
+
+To enable elasticity, microservices must be mobile and addressable. A Service Registry supports these requirements by maintaining an up-to-date list of available microservices that includes the host and port numbers for each instance. As load changes, the system can spawn or destroy instances in any location and continue to satisfy requests because the Service Registry maintains the addresses of all active instances for each microservice. 
```

Comment:
> we should spell things out a bit more.

+1 

The suggestion is great. Will merge with the current wording ( @rstento helped with that too :-) )